unofficial_flocker_tools/config.py

```diff
@@ -143,6 +144,36 @@ def main(reactor, args):
 
     yield gatherResults(deferreds)
 
+    if c.config["os"] == "ubuntu":
+        if len(sys.argv) > 2:
+            if sys.argv[2] == "--swarm":
+                clusterid = c.runSSH(c.config["control_node"], """
+docker run swarm create""").strip()
+                log("Created Swarm ID")
+                for node in c.config["agent_nodes"]:
+                    d = c.runSSHAsync(node['public'], """
+service docker stop
+docker daemon -H unix:///var/run/docker.sock -H tcp://0.0.0.0:2375 >> /tmp/dockerlogs 2>&1 &
```

Comment:
Let's regex `/etc/default/docker` instead. This is an ephemeral, unreliable way of starting docker with some args.